src/legacy/core_plugins/kibana/server/tutorials/apm/index.js

```diff
@@ -91,9 +91,7 @@ It allows you to monitor the performance of thousands of applications in real ti
     savedObjectsInstallMsg: i18n.translate(
       'kbn.server.tutorials.apm.specProvider.savedObjectsInstallMsg',
       {
-        defaultMessage:
-          'Load index pattern, visualizations, and pre-defined dashboards. \
-An index pattern is required for some features in the APM UI.',
+        defaultMessage: 'An APM index pattern is required for some features in the APM UI.',
```

Comment:
would it be helpful to list some affected features?

Reply:
I'd prefer to keep this as close to what was recommended in the ticket as possible for now.